package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axlair/jupyterlab_vim",
-  "version": "0.13.4",
+  "version": "0.13.5-rc.2",
```

Comment:
What's the purpose of adding `rc.2`? Won't that make it so that `pip install` doesn't pick up this version?

Reply:
We can remove that. I accidently left that during my test of publishing this to an internal registry